src/firebolt/service/V2/engine.py

```diff
@@ -115,6 +115,18 @@ def get_many(
             ]
             return [Engine._from_dict(_dict, self) for _dict in dicts]
 
+    @staticmethod
+    def _format_engine_parameter(
+        value: Union[str, int, EngineType, InstanceType, WarmupMethod]
```

Comment:
nit: we should include `float` here

Reply:
I would better not allow float. This would remove any confusion, there is no need to use it anyways